src/features/ShowHelp.ts

```diff
@@ -6,34 +6,42 @@ import vscode = require("vscode");
 import { LanguageClient, NotificationType, RequestType } from "vscode-languageclient";
 import { IFeature } from "../feature";
 
+// TODO: remove ShowOnlineHelpRequest in v2
```

Comment:
Do actually need to keep this type around if we only use the new message? We need to keep it in PSES, but maybe not here?

Reply:
Agreed.  AFAICT the ShowOnlineHelpRequestType is not used anymore in this file.